### DISCUSSION TITLE
Embed iframe  design tab

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#1364] [Help] [answered]
when I use the "Embed" function in the design tab and input my iframe info nothing happens. when previewing the page its just blank.  is this function broken? or am I doing it wrong?

Thank you

**Comment by joebudi** [comment]
Hey AndrewJRamay. Could you share a screenshot?

**Reply by KernOil** [reply]
![image](https://user-images.githubusercontent.com/80058238/115472321-7f011d80-a1ee-11eb-8036-50e0f3d2cab5.png

this is all i get when trying to put in an iframe. i have to put one } at the end for it to do anything if i put it in {{}} or leave it alone it dose nothing.

<iframe style="background: #FFFFFF;border: none;border-radius: 2px;box-shadow: 0 2px 10px 0 rgba(70, 76, 79, .2);" width="640" height="480" src=""></iframe>}

**Reply by KernOil** [reply]
![image](https://user-images.githubusercontent.com/80058238/115472518-e1f2b480-a1ee-11eb-9950-b36730143d2a.png)

**Reply by joebudi** [reply]
Interesting.

Try adding px to the end of 640 and 480 - 640px and 480px.

**Reply by KernOil** [reply]
That did the trick! thank you sir!

**Reply by gauravpay3ds** [reply]
> ![image](https://user-images.githubusercontent.com/80058238/115472518-e1f2b480-a1ee-11eb-9950-b36730143d2a.png)

can i get the code reference of embed component ? I have tried but only view is working no java script is working

**Comment by aptkingston** [comment, marked as answer]
The way embeds are implemented, you'll need to use the actual builder settings to control the size of your iframe, rather than styling the iframe itself. If you select your embed and go to the "Design" tab, you can set a width and height there.

Here's what it'll look like when you do that:
![image](https://user-images.githubusercontent.com/9075550/115518051-aeac3600-a27f-11eb-896a-7b20747eee02.png)
